src/models/taskList/postholderDetails.task.js

```diff
@@ -0,0 +1,99 @@
+'use strict'
+
+const Handlebars = require('handlebars')
+
+const BaseTask = require('./base.task')
+const LoggingService = require('../../services/logging.service')
+const CryptoService = require('../../services/crypto.service')
+const ContactDetail = require('../../models/contactDetail.model')
+const Address = require('../../persistence/entities/address.entity')
+const { POSTHOLDER_CONTACT_DETAILS } = require('../../dynamics').AddressTypes
+
+module.exports = class PostholderDetails extends BaseTask {
```

Comment:
Should this and partnerDetails.task.js extend a common base class as they are almost identical?